WalletWasabi.Fluent/Controls/Dialog.axaml.cs

```diff
@@ -73,6 +75,8 @@ public class Dialog : ContentControl
 	public static readonly StyledProperty<bool> ShowAlertProperty =
 		AvaloniaProperty.Register<Dialog, bool>(nameof(ShowAlert));
 
+	private static readonly Stack<Dialog> Open = new ();
```

Comment:
it should have a better name for understanding. Something like `AllOpenedDialogStack`

Reply:
Renamed